azure_monitor/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## Unreleased
+
+## 0.1a.0
+Released 2019-11-05
```

Comment:
Did you released this today?

Reply:
I'm planning on releasing today.

Reply:
Actually on second thought, I should change this to unreleased until I actually release it.